odoo/sql_db.py

```diff
@@ -82,6 +90,21 @@ def clear_env(cr):
 sql_counter = 0
 
 
+QUERY_CATCH_VERBOSE = os.environ.get('ODOO_QUERY_USERS_VERBOSE')
+QUERIES_TO_CATCH = os.environ.get('ODOO_QUERY_TO_CATCH') or [
+    'SELECT "res_users"."id" as "id", "res_users"."partner_id" as "partner_id", "res_users"."login" as "login", "res_users"."signature" as "signature", "res_users"."active" as "active", "res_users"."action_id" as "action_id", "res_users"."share" as "share", "res_users"."company_id" as "company_id", "res_users"."create_uid" as "create_uid", "res_users"."create_date" as "create_date", "res_users"."write_uid" as "write_uid", "res_users"."write_date" as "write_date", "res_users"."notification_type" as "notification_type", "res_users"."karma" as "karma", "res_users"."rank_id" as "rank_id", "res_users"."next_rank_id" as "next_rank_id", "res_users"."odoobot_state" as "odoobot_state", "res_users"."odoobot_failed" as "odoobot_failed", "res_users"."sale_team_id" as "sale_team_id", "res_users"."oauth_provider_id" as "oauth_provider_id", "res_users"."oauth_uid" as "oauth_uid", "res_users"."oauth_access_token" as "oauth_access_token", "res_users"."google_cal_account_id" as "google_cal_account_id", "res_users"."subscribe_job" as "subscribe_job", "res_users"."last_seen_phone_call" as "last_seen_phone_call", "res_users"."target_sales_won" as "target_sales_won", "res_users"."target_sales_done" as "target_sales_done", "res_users"."helpdesk_target_closed" as "helpdesk_target_closed", "res_users"."helpdesk_target_rating" as "helpdesk_target_rating", "res_users"."helpdesk_target_success" as "helpdesk_target_success", "res_users"."livechat_username" as "livechat_username", "res_users"."website_id" as "website_id", "res_users"."tiledesk_user_id" as "tiledesk_user_id", "res_users"."tiledesk_token" as "tiledesk_token", "res_users"."target_sales_invoiced" as "target_sales_invoiced" FROM "res_users" WHERE "res_users".id IN (',
+    'SELECT "res_country".id FROM "res_country" LEFT JOIN "ir_translation" AS "res_country__name" ON ("res_country"."id" = "res_country__name"."res_id" AND "res_country__name"."type" = \'model\' AND "res_country__name"."name" = \'res.country,name\' AND "res_country__name"."lang" = \'en_US\' AND "res_country__name"."value" != '') WHERE ("res_country"."code" = \'US\') ORDER BY COALESCE("res_country__name"."value", "res_country"."name") LIMIT 1;',
+    'SELECT min("ir_model_fields_tracking".id) AS id, count("ir_model_fields_tracking".id) AS "field_name_count", "ir_model_fields_tracking"."field_name" as "field_name" FROM "ir_model_fields_tracking" WHERE ("ir_model_fields_tracking"."model_name" = \'product.product\') GROUP BY "ir_model_fields_tracking"."field_name" ORDER BY "field_name";',
+    'SELECT "res_partner".id FROM "res_partner" WHERE ((("res_partner"."active" = true) AND "res_partner"."parent_id" IS NULL) AND (FALSE OR ',
```

Comment:
I think we know where this comes... but not sure

We need to check

For now it is enough with the first one `res_users.id IN (...`